Library/Homebrew/cmd/update.sh

```diff
@@ -785,11 +778,16 @@ EOS
       JSON_URLS+=("${HOMEBREW_API_DEFAULT_DOMAIN}/${formula_or_cask}.json")
       for json_url in "${JSON_URLS[@]}"
       do
+        time_cond=()
+        if [[ -s "${HOMEBREW_CACHE}/api/${formula_or_cask}.json" ]]
+        then
+          time_cond=("--time-cond" "${HOMEBREW_CACHE}/api/${formula_or_cask}.json")
+        fi
         curl \
           "${CURL_DISABLE_CURLRC_ARGS[@]}" \
-          --fail --compressed --silent --max-time 30 \
+          --fail --compressed --silent --speed-limit 100 --speed-time 30 \
```

Comment:
I think it might be worth pulling these values into something in `brew.sh` that can be shared through environment variables with the Ruby-land too.

Reply:
Yes, though that applies to all args here and is a pre-existing issue.